pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/RealtimeSegmentImpl.java

```diff
@@ -50,253 +43,222 @@
 
 
 public class RealtimeSegmentImpl implements RealtimeSegment {
-  private final Logger LOGGER;
-  public static final int[] EMPTY_DICTIONARY_IDS_ARRAY = new int[0];
+  // For multi-valued column, forward-index.
+  // Maximum number of multi-values per row. We assert on this.
+  private static final int MAX_MULTI_VALUES_PER_ROW = 1000;
 
-  private static final String MMGR_DICTIONARY_SUFFIX = ".dict";
```

Comment:
these suffixes help us look over the logs and see which part allocates memory. so please add these back.